<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComClass</class>
 <widget class="QDialog" name="ComClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>541</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="allBoxLayout">
    <item>
     <widget class="QGroupBox" name="enviarBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Texto a enviar:</string>
      </property>
      <widget class="QTextEdit" name="enviarText">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>511</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="enviarButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>91</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Enviar</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="rcvBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Texto recivido:</string>
      </property>
      <widget class="QTextBrowser" name="rcvText">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>521</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="conexionGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Conexion</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="puertoLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>puerto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="puertosComboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="ActualizarButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>actualizar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="conectarButton">
       <property name="text">
        <string>conectar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>589</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="statusLayout"/>
  </widget>
 </widget>
 <tabstops>
  <tabstop>puertosComboBox</tabstop>
  <tabstop>ActualizarButton</tabstop>
  <tabstop>conectarButton</tabstop>
  <tabstop>enviarText</tabstop>
  <tabstop>enviarButton</tabstop>
  <tabstop>rcvText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
